datacatalog/pkg/config/config_flags.go

```diff
@@ -52,6 +52,7 @@ func (cfg Config) GetPFlagSet(prefix string) *pflag.FlagSet {
 	cmdFlags := pflag.NewFlagSet("Config", pflag.ExitOnError)
 	cmdFlags.Int(fmt.Sprintf("%v%v", prefix, "grpcPort"), defaultConfig.GrpcPort, "On which grpc port to serve Catalog")
 	cmdFlags.Bool(fmt.Sprintf("%v%v", prefix, "grpcServerReflection"), defaultConfig.GrpcServerReflection, "Enable GRPC Server Reflection")
+	cmdFlags.Int(fmt.Sprintf("%v%v", prefix, "grpcMaxRecvMsgSizeMBs"), defaultConfig.GrpcMaxRecvMsgSizeMBs, "The max receive message size; if unset defaults to gRPC server default value")
```

Comment:
I actually modified this manually because `make generate` didn't work for me.
```
Installing github.com/vektra/mockery/v2@v2.40.3
go: github.com/vektra/mockery/v2@v2.40.3 (in github.com/vektra/mockery/v2@v2.40.3): go.mod:5: unknown directive: toolchain
```

Reply:
I think something is not working well with goenv.

Reply:
I had to use `goenv shell`.

Reply:
Proposed a minor fix in #6314